app/views/pages/home.html.haml

```diff
@@ -13,86 +13,43 @@
-          %p= link_to 'See how your service can use registers', using_registers_path
-        .column-one-half
-          = image_tag 'get-records.png'
-
-    %section.content-section.content-section--with-top-border
-      .grid-row
-        .column-one-half
-          %h2.heading-medium.content-section__title Why your service should use registers
-          %p Your service can benefit from using registers because they’re:
+          %h2.heading-medium.content-section__title GOV.UK Registers helps you to:
           %ul.list.list-bullet
-            %li accurate – you can trust the data and it’s updated for you
-            %li consistent – all services use the same data
-            %li approved – it’s the government’s official data
-          %p= link_to 'See all the data you can use', registers_path
-
+            %li build and maintain your service with accurate data that is updated for you
+            %li reduce the time and cost of sourcing data from across government
+            %li easily share data with other government departments
         .column-one-half
           = image_tag 'get-data.png'
 
-    %section.content-section.content-section--with-top-border
-      .grid-row
-        .column-full
-          %h2.heading-medium.content-section__title Registers in numbers
-      .grid-row
-        .column-one-third
-          .data
-            = link_to registers_path do
-              %span.data-item.bold-xxlarge= @ready_registers.count
-              %span.data-item.bold-xsmall registers you can use
-        .column-one-third
-          .data
-            = link_to registers_in_progress_path do
-              %span.data-item.bold-xxlarge= @upcoming_registers.count
-              %span.data-item.bold-xsmall upcoming registers
-        .column-one-third
-          .data
-            = link_to services_using_registers_path do
-              %span.data-item.bold-xxlarge 13
-              %span.data-item.bold-xsmall government services using registers
-
     %section.content-section.content-section--with-top-border
       .grid-row
         .column-one-half
-          %h2.heading-medium.content-section__title Combining registers
-          %p You can easily combine data from more than one register. This can be useful if you need data from different organisations.
-          %p For example, you might use data from the #{link_to 'country register', '/registers/country'} as well as the #{link_to 'territory register', '/registers/territory'} to build a feature that allows your users to find their location.
-          %p= link_to 'Read more about combining registers', combining_registers_path
-
+          %h2.heading-medium.content-section__title How registers work
+          %p Registers are live lists of information that are managed and approved by government. They are the most reliable lists of thier kind.
+          %p Each register:
+          %ul.list.list-bullet
+            %li can be accessed and used freely by anyone
```

Comment:
erm, mixed messages if at the same time we are encouraging API key registration? @mpwoods @bahmady @conordelahunty  🤷‍♂️